src/smdba/postgresqlgate.py

```diff
@@ -155,9 +158,13 @@ def estimate(self):
         # No more than 1GB
         self.config['maintenance_work_mem'] = self.to_mb(self.br((mem / 0x10) > megabytes and megabytes or mem / 0x10))
 
-        self.config['checkpoint_segments'] = 8
+        if "postgresql94" in os.path.realpath(PG_VERSION):
+            self.config['checkpoint_segments'] = 8
+        else:
+            self.config['max_wal_size'] = 384
```

Comment:
So, assuming `PG_VERSION` is now set to `[9, 4, 12]` as per above comment, I would:
```python
if PG_VERSION < [9, 6, 0]:
    self.config['.....'] = ...
else:
    ...
```